config/routes.rb

```diff
@@ -5,6 +5,8 @@
   	resources :bookmarks
   end
 
+  post :incoming, to: 'incoming#create'
```

Comment:
I'm not sure what was causing the 404 issue, but a while later it resent and worked. I had added that from the tutorial so I think it was something wrong with the incoming_controller.rb